src/main/java/FakePlayer.java

```diff
@@ -2,21 +2,21 @@
 
```

Comment:
Is `FakePlayer` meant to be in the main source package?

Reply:
Thanks for spotting this :) I'll move it